### PR TITLE
Fixed proofit404/anaconda-mode/issues/132

### PR DESCRIPTION
**Pull request by KenetJervet**
Hi. This is a new attempt. Please take a look. :dancer:

### CODE REVIEW

**Comment by proofit404** [comment]
Hello. Thanks for you contribution. Rebase all changes into one commit.

File: anaconda-mode.el [hunk 1]
```diff
@@ -632,10 +632,15 @@ PRESENTER is the function used to format buffer content."
   "Find DEFINITION file other window, go to DEFINITION point."
   (anaconda-mode-find-file-generic definition 'find-file-other-window))
 
-(defvar-local anaconda-mode-go-back-definition nil
+(defun anaconda-mode-find-file-no-record-definition (definition)
+  "Find DEFINITION file, go to DEFINITION point (without recording in the go-back stack)"
+  (anaconda-mode-find-file-generic definition 'find-file t)
+  )
```
Comment (proofit404) [inline anaconda-mode.el]: Never do that again :D

Comment (KenetJervet) [inline anaconda-mode.el]: Sorry I didn't get it?

**Comment by KenetJervet** [comment]
I'm done with rebasing :)

File: anaconda-mode.el (hunk 1 above)
Comment (proofit404) [inline anaconda-mode.el]: Don't add new line before `)`. Always keep closing brackets on there same line. This is common style for all languages in the Lisp family ;)

**Comment by proofit404** [comment]
Thanks for your contributions and for your patience.

**Comment by KenetJervet** [comment]
@proofit404 You're welcome! I should be sorry for the poor quality of my contribution. I seldom contribute code. Will do better next time :D

**Pull request merged**